app/serializers/job_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class JobSerializer < ActiveModel::Serializer
   attributes :id, :title, :description, :url, :location, :posted_date, :remote
-  has_one :company
+  has_one :company, serializer: CompanyForJobsSerializer
```

Comment:
Question: could you just use the one jobs serializer if you added the location conditional to it? are they otherwise the same? if the jobs it's sorting through in the database are the same, might any job come without a location, in which case- you found a bug in the old serializer and can just use one for all jobs! yay!